sway/desktop/output.c

```diff
@@ -833,13 +834,22 @@ static void handle_present(struct wl_listener *listener, void *data) {
 	output->refresh_nsec = output_event->refresh;
 }
 
+static unsigned int last_headless_num = 0;
+
 void handle_new_output(struct wl_listener *listener, void *data) {
 	struct sway_server *server = wl_container_of(listener, server, new_output);
 	struct wlr_output *wlr_output = data;
+
 	if (wlr_output == root->fallback_output->wlr_output) {
 		return;
 	}
 
+	if (wlr_output_is_headless(wlr_output)) {
```

Comment:
Isnt this handleded by [36a2b19485ad299ee0039eb97c0f688d68205539] ?(https://gitlab.freedesktop.org/wlroots/wlroots/-/commit/36a2b19485ad299ee0039eb97c0f688d68205539)

Reply:
What do you mean?

That wlroots commit only allows compositors to set a custom output name. It doesn't change the fact that the name wlroots picks for the fallback output is "HEADLESS-1", which breaks users config files.

Reply:
Oh right, the fallback input will be named `FALLBACK` after it has been named `HEADLESS-1` by wlroots, thus messing the order.